platform/broadcom/docker-syncd-brcm-dnx/saidump.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
```

Comment:
can you move this file to sairedis repo?

Reply:
Yes. The sairedis is using c++. I need to re-write saidump.sh from shell to C++.

Reply:
there are script in syncd https://github.com/sonic-net/sonic-sairedis/tree/master/syncd/scripts

Reply:
Thanks. I will move saidump.sh into this folder.

Reply:
@lguohan , @mlok-nokia , please review again. And a new PR was created as below.
https://github.com/sonic-net/sonic-sairedis/pull/1298